src/master-playlist-controller.js

```diff
@@ -429,8 +402,17 @@ export class MasterPlaylistController extends videojs.EventTarget {
       let updatedPlaylist = this.masterPlaylistLoader_.media();
 
       if (!updatedPlaylist) {
-        // select the initial variant
-        this.initialMedia_ = this.selectPlaylist();
+        let selectedMedia;
```

Comment:
We should probably dig into why this was happening since I don't think it should cause the test to break